lessons/Early-Math/identity-property/langs/en-us/pages/true-or-false-variables.adoc

```diff
@@ -0,0 +1,62 @@
+= True or False? (Identity Property with Variables)
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+td > .content > .paragraph > * { vertical-align: middle; }
+</style>
+++++
+
+Is the equation represented by the two Circles true or false? Explain why, and then describe what you would change to make the _false_ equations _true_.
+
+[.FillVerticalSpace,cols="^.^1a,^.^8a,^.^5a", stripes="none", options="header"]
+|===
+|	 | Circles																	   |
+True or False? Explain
+
+
+| @n
+| @show{(coe '(* m 1))}
+@math{=}
+@show{(coe '(* m (- 4 3)))}
+|
+
+
+| @n
+| @show{(coe '(* (+ g h) (/ 30 30)))}
+@math{=}
+@show{(coe '(* (+ h g) (/ 523 523)))}
+|
+
+
+| @n
+| @show{(coe '(+ (* h (* t q)) 0))}
+@math{=}
+@show{(coe '(+ 0 (* q (* t h))))}
+|
+
+
+
+| @n
+| @show{(coe '(+ (- f h) (- t t)))}
+@math{=}
+@show{(coe '(+ (- h f) (- k k)))}
+|
+
+
+| @n
+| @show{(coe '(* (/ 35 b) (/ k k)))}
+@math{=}
+@show{(coe '(/ 35 b))}
+|
+
+
+
+| @n
+| @show{(coe '(- 76.42 (- (* 5 k) (* 5 k))))}
+@math{=}
+@show{(coe '(- 76.42 (- (/ w 3) (/ w 3))))}
+|
+
+
```

Comment:
While I recognize that you're showing equivalence here, to me only number 5 feels like it's representing the identity property